doc/source/tutorials.rst

```diff
@@ -0,0 +1,11 @@
+Tutorials
+---------
+
+Overview
+========
```

Comment:
This extra level is probably not needed if we only have 2 tutorials. Also, what about putting them into tiles rather than a list? That could be more visually appealing, especially if you make a little picture to go with each tile.
```suggestion
```